server_environment/__manifest__.py

```diff
@@ -21,7 +21,7 @@
 
 {
     "name": "server configuration environment files",
-    "version": "9.0.1.2.0",
+    "version": "10.0.1.2.0",
```

Comment:
my personal taste goes to keeping the last 3 numbers, since this could help checking if things have changed in the Odoo_versionN-1 of the module. But I've repeatedly been asked to reset these...